test/AspNet.Security.OAuth.Providers.Tests/Ebay/EbayTests.cs

```diff
@@ -0,0 +1,50 @@
+﻿/*
+ * Licensed under the Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
+ * See https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers
+ * for more information concerning the license and the contributors participating to this project.
+ */
+
+using System;
+using System.Security.Claims;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authentication;
+using Microsoft.Extensions.DependencyInjection;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace AspNet.Security.OAuth.Ebay
+{
+    public class EbayTests : OAuthTests<EbayAuthenticationOptions>
+    {
+        public EbayTests(ITestOutputHelper outputHelper)
+        {
+            OutputHelper = outputHelper;
+        }
+
+        public override string DefaultScheme => EbayAuthenticationDefaults.AuthenticationScheme;
+
+        protected internal override void RegisterAuthentication(AuthenticationBuilder builder)
+        {
+            builder.AddEbay(options =>
+            {
+                options.RuName = $"localhost{options.CallbackPath}";
```

Comment:
Is it correct this doesn't contain a scheme? It looks odd. For example, `https://localhost/signin-ebay`.

Reply:
I have not investigated, but it seems that the pattern of the initial request is maintained and this seems logical to me. However, I tried adding the pattern and it is respected without being duplicated.